api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -126,6 +126,10 @@ type DRPlacementControlSpec struct {
 
 	// Action is either Failover or Relocate operation
 	Action DRAction `json:"action,omitempty"`
+
+	// KubeObjectProtection fields to copy to VolumeReplicationGroup
```

Comment:
(nit) this text appears in auto-generated go.dev documentation.  DRPC users may not be familiar with VolumeReplicationGroup.  I suggest removing it